tests/end_to_end/test_docker_monitoring/data/test_cases/cases_test_docker_monitoring.yaml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
The file name should be: `cases_docker_monitoring.yaml`

Reply:
Done in [ced7d89](https://github.com/wazuh/wazuh-qa/pull/2948/commits/ced7d8990b0043155fe47e9532b36bae152b0bc0), https://github.com/wazuh/wazuh-qa/pull/2948/commits/3baa409773dec6b57996240c9dfbe89bdaf83918, and https://github.com/wazuh/wazuh-qa/pull/2948/commits/b50405915f4d3c5bea22fd3fefa676e5d3d8bb9d